crest/Assets/Crest/Crest/Scripts/Helpers/OceanDebugGUI.cs

```diff
@@ -216,5 +216,16 @@ void ToggleGUI()
         {
             _guiVisible = !_guiVisible;
         }
+
+#if UNITY_2019_3_OR_NEWER
+        [RuntimeInitializeOnLoadMethod(RuntimeInitializeLoadType.SubsystemRegistration)]
+#endif
+        static void InitStatics()
+        {
+            // Init here from 2019.3 onwards
+            s_drawTargets = new Dictionary<System.Type, bool>();
```

Comment:
I would keep the dictionaries as readonly and just clear them.

Reply:
Ok ill do it that way